jboss/container/wildfly/launch-config/os/added/launch/launch-common.sh

```diff
@@ -115,4 +115,19 @@ function splitAttributesStringIntoLines() {
   local temp
   temp=$(echo $input | sed "s|\" ${attribute_name}=\"|\" \n${attribute_name}=\"|g" | awk -F "\"" '{print $2}')
   echo "${temp}"
-}
\ No newline at end of file
+}
+
+# Find the ipv4 address of the host
+# The host could have 1 or more ipv4 and/or ipv6 addresses
+# For this function we need to return a single ipv4 address
+function get_host_ipv4() {
+ mapfile < <(awk '/32 host/ { print f } {f=$2}' <<< "$(</proc/net/fib_trie)" | sort -n | uniq | grep -v '127.0.0.')
```

Comment:
I tried locally and have a syntax error for this line. 
Could you add some comments to explain what the command is doing, it is not obvious when looking at the command. Specially the awk part.
Is the variable named MAPFILE?

Reply:
MAPFILE is the default variable name used by Bash when using the builtin function mapfile. https://www.geeksforgeeks.org/mapfile-command-in-linux-with-examples/

For the awk part I actually just copied over from @luck3y. It is a awk block pattern '/pattern/ {command}'
the /32 host/ block-pattern filters out lines containing 32 or host from the output of /proc/net/fib_trie